controllers/concerts.js

```diff
@@ -51,7 +51,8 @@ const createConcert = async (req, res) => {
                                                            req.body.ticket_amount, // Create the tickets_available from the ticket_amount
                                                            picture
     );
-    res.redirect("/test_indexConcert.html");
+    // here 
```

Comment:
why the here?

Reply:
בשביל עמית היא צריכה להכניס שם פונקציונאליות של פייסבוק